### ISSUE TITLE
"Are you sure you are in the right league" error

### ISSUE THREAD

**Issue by Blaster08** [Stickymaddness/Procurement#536]
I'm able to put in my email and password. It starts logging in and then says "Error getting items for league, are you sure your league settings are correct?".

Thanks for the help.

[DebugInfo.zip](https://github.com/Stickymaddness/Procurement/files/167490/DebugInfo.zip)

**Comment by Stickymaddness** [comment]
Have you used Procurement before? 

According to the error message, you don't have any leagues associated with your account.

**Comment by Blaster08** [comment]
No, this is my first time using it.

**Comment by Stickymaddness** [comment]
Alright, try logging in using sessionID. 

Take a look at this video guide if you need help logging in with sessionID: https://www.youtube.com/watch?v=nQFcQTW8ju0

**Comment by Blaster08** [comment]
I'm getting the same error message.

**Comment by Blaster08** [comment]
Something I've noticed that is slightly different I'm not sure if it is relevant or not, the video has a box to enter an Account Name. I do not have that box. I only have a box for Alias and Session ID.

**Comment by Stickymaddness** [comment]
Yup that is correct - Account Name was removed and is automatically retrieved by Procurement. 

Are you a steam player?

I would contact GGG support and tell them that the API is not returning any leagues/items for your account, they are normally pretty quick in sorting it out. After that you should be able to login no problem.

**Comment by Blaster08** [comment]
Yes, I am a steam player.

I will contact them. Thank you for your help.

**Comment by Blaster08** [comment]
I've resolved the issue. I did not first notice the "log in through steam" button on the forums. I've logged in that way and used the session ID and everything works well now. Thanks!
